hat/settings.py

```diff
@@ -519,6 +519,16 @@ def sentry_tracer_sampler(sampling_context):
 THEME_SECONDARY_COLOR = os.environ.get("THEME_SECONDARY_COLOR", "#0066CC")
 THEME_PRIMARY_BACKGROUND_COLOR = os.environ.get("THEME_PRIMARY_BACKGROUND_COLOR", "#F5F5F5")
 SHOW_NAME_WITH_LOGO = os.environ.get("SHOW_NAME_WITH_LOGO", "yes")
+# OpenHexa API url
```

Comment:
I think that this should live in the Config model, not in the server settings. Changing the OpenHexa token should not require to redeploy the server, I think.